Automattic/wp-calypso
# Datacenter Picker: Incorporate illustrations

See https://github.com/Automattic/dotcom-forge/issues/1237

Design discussion in p9Jlb4-5Xy-p2

## Proposed Changes

Incorporates illustrations into the datacenter picker UI:

<img width="810" alt="image" src="https://user-images.githubusercontent.com/36432/204539379-7f82ded4-37f8-4002-8b1d-e2cdaf301bcd.png">

## Testing Instructions

1. Create a new WordPress.com site with a business plan.
2. Navigate to 'Hosting Configuration' and click 'Activate'.
3. Verify the `<EligibilityWarnings>` component displays with the Datacenter Picker UI.
4. Verify the Datacenter Picker UI is usable, accessible, and looks good in the browsers we support.
5. If you want to recreate the 'no eligibility warnings' state, map a domain to your site and then go back to 'Hosting Configuration' -> 'Activate'.
